cmd/konvoy-image-wrapper/cmd/wrapper.go

```diff
@@ -503,7 +515,10 @@ func (r *Runner) Run(args []string) error {
 	if err != nil {
 		return fmt.Errorf("failed to make temp %w", err)
 	}
-	defer os.RemoveAll(r.tempDir)
+	defer func() {
+		fmt.Println("Removing temporary directory", r.tempDir)
+		os.RemoveAll(r.tempDir)
```

Comment:
For some reason this command errors out because `${tmp_dir}/.aws/ ` and `{tmp_dir}/.ssh/` are owned by root.